### ISSUE TITLE
sp_BlitzFirst output table's CheckDate can be improved

### ISSUE THREAD

**Issue by ben9tamu** [BrentOzarULTD/SQL-Server-First-Responder-Kit#3310]
**Version of the script**
Look in the stored procedure, and it'll have a version date & number near the top. Put that in here. If it's not the current version (dated in the last month), then upgrade to the current version and test that before reporting a bug - we fix a lot of stuff in each new build. We'll flat out close bug reports for older builds.

version 8.15, 20230613

**What is the current behavior?**

In sp_Blitz, when wait stats are output to a table, CheckDate column is set to @StartSampleTime. It should be @FinishSampleTime to be consistent with the result in result pane.

**If the current behavior is a bug, please provide the steps to reproduce.**

Here's an example.
```
EXEC dba.dbo.sp_BlitzFirst @SinceStartup = 1, 
    @OutputServerName = 'devsql', 
    @OutputDatabaseName = 'dbatools', 
    @OutputSchemaName = 'dbo', 
    @OutputTableNameWaitStats = 'BlitzWait',
    @OutputTableNameFileStats = 'BlitzFile',
    @OutputTableNamePerfmonStats = 'BlitzPerf'

USE DBATools
SELECT * FROM dbo.BlitzWait
SELECT * FROM dbo.BlitzFile
SELECT * FROM dbo.BlitzPerf
```

You will see that the tables has @StartSampleTime in CheckDate columns.

**What is the expected behavior?**

CheckDate should be the same as @FinishSampleTime. 
In other words, it should be the same as "Sample Ended" column for wait stats, "Sample Time" column for file stats, and "LastSampleTime" in perfmon stats, when sp_BlitzFirst is run without Output*** parameters.

**Which versions of SQL Server and which OS are affected by this issue? Did this work in previous versions of our procedures?**
This is the same in all versions.

**Comment by BrentOzar** [comment]
The output to table isn't really designed to be compared to what's on the screen. Either run it interactively, or log it to table, but it's not really for both at the same time.
